### ISSUE TITLE
Retweets of quote tweets should include quote

### ISSUE THREAD

**Issue by klausi** [klausi/mastodon-twitter-sync#16]
If I retweet a quote tweet on Twitter then I get on Mastodon:
```
RT example: tweet text <twitter link to quote>
```
We should include the quote as we do if I directly quote a tweet:
```
RT example: tweet text
QT quote text
```

**Comment by bajtos** [comment]
@klausi I think this issue can be closed as done via 41946d1d3?

**Comment by klausi** [comment]
No, the current quote tweet behavior only works for your own tweets.

This issue is about supporting retweets. If a retweet is a quote tweet then we should also include the quoted text.
